integration_tests/dbt_project.yml

```diff
@@ -88,6 +88,3 @@ seeds:
     team_data:
       +column_types:
         id: "{{ 'int64' if target.type == 'bigquery' else 'bigint' }}"
-clean-targets:
-- target
-- dbt_modules
```

Comment:
Removed these add the clean target has changed to dbt_packages and that is used by default. Therefore there is no need to include them anymore. It also threw a weird warning when regenerating the docs.